src/app/components/tooltip/tooltip.ts

```diff
@@ -169,20 +173,27 @@ export class Tooltip implements AfterViewInit, OnDestroy {
     ngAfterViewInit() {
         if (isPlatformBrowser(this.platformId)) {
             this.zone.runOutsideAngular(() => {
-                if (this.getOption('tooltipEvent') === 'hover') {
-                    this.mouseEnterListener = this.onMouseEnter.bind(this);
-                    this.mouseLeaveListener = this.onMouseLeave.bind(this);
-                    this.clickListener = this.onInputClick.bind(this);
-                    this.el.nativeElement.addEventListener('mouseenter', this.mouseEnterListener);
-                    this.el.nativeElement.addEventListener('click', this.clickListener);
-                    this.el.nativeElement.addEventListener('mouseleave', this.mouseLeaveListener);
-                } else if (this.getOption('tooltipEvent') === 'focus') {
-                    this.focusListener = this.onFocus.bind(this);
-                    this.blurListener = this.onBlur.bind(this);
-
-                    let target = this.getTarget(this.el.nativeElement);
-                    target.addEventListener('focus', this.focusListener);
-                    target.addEventListener('blur', this.blurListener);
+                if (this.isMobile()) {
+                    this.touchStartListener = this.onTouchStart.bind(this);
+                    this.touchEndListener = this.onTouchEnd.bind(this);
+                    this.el.nativeElement.addEventListener('touchstart', this.touchStartListener);
+                    this.el.nativeElement.addEventListener('touchend', this.touchEndListener);
+                } else {
+                    if (this.getOption('tooltipEvent') === 'hover') {
```

Comment:
I liked what you did with `isMobile` can you do same here? Like `isHoverEvent` or so?